src/Components/Patient/ManagePatients.tsx

```diff
@@ -98,6 +98,8 @@ export const PatientManager = (props: any) => {
     const params = {
       csv: true,
       facility: facilityId,
+      disease_status: qParams.disease_status
```

Comment:
No need for name and phone_number ? @bodhish  @tomahawk-pilot

Reply:
It's more like a search than a filter function. I guess we could avoid it for now.